tests/integration/Acme/UserTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
Shall we keep this file around anymore? Looks like maybe it was just for testing locally.

Reply:
I'll leave it as a sample for now, will be removed once more tests are added.